patches/common/v8/.patches.yaml

```diff
@@ -64,6 +56,11 @@ patches:
   owners: codebytere
   file: cherry_pick_ac0fe8ec.patch
   description: null
+-
+  owners: alexeykuzmin,alespergl
+  file: export_platform.patch
```

Comment:
I'm pretty sure you moved the patch closer to the end of the list intentionally )
What's the reason to do that?

Reply:
Yes, because I generated the patch from a state where the previous patches are already applied. 😉